store-implementation/accumulo-store/README.md

```diff
@@ -55,13 +55,13 @@ Gaffer's `AccumuloStore` is particularly well-suited to graphs where the propert
 Accumulo set up
 -----------------------------------------------
 
-Gaffer has been extensively tested with Accumulo version 1.7.2. It should work with any of the 1.7.* versions of Accumulo.
+Gaffer has been extensively tested with Accumulo version 1.8.1. It is recommended to use this version, although it should work with any of the 1.7.* versions of Accumulo as well.
```

Comment:
This should say:

should work with any of the 1.8.* versions of Accumulo as well